### ISSUE TITLE
Login error even though I'm already logged in.

### ISSUE THREAD

**Issue by Dexxxxx** [adamb70/CSGO-Market-Float-Finder#17]
So when I start it up it asks for username, password and mobile auth code.

After entering the code one of two things happen: A message pops up saying the code is incorrect, or a message pops up asking to enter the auth code again. After entering the same code again it logs in successfully, but as soon as I click start I get an error that I'm not logged in and should restart the program.

This used to happen only occasionally, but now it happens every time I try to log in so it is literally unusable.

And no, I'm not running CSGO in the background.

**Comment by SirMerideo** [comment]
Same here...

**Comment by SirMerideo** [comment]
Did you check if you had "http://" in your link? I forgot chrome generates it when you copy the link, that was the problem for me. I just had to delete it.

**Comment by Dexxxxx** [comment]
It works fine for me with or without "http://" in the link. My problem is with the "Start" button, not the "Retrieve Items" button. I can always retrieve the items, but sometimes when I click on start I get the login error and have to restart the program.

I just got it to work again without getting the login error, but it still doesn't work 100% of the time like it should so there is obviously still something wrong somewhere.

**Comment by SirMerideo** [comment]
Yea but I'm talking about the Start button, it doesn't really make sense, but that's how I fixed it

**Comment by Dexxxxx** [comment]
After closing the program I am now unable to log in again. Still get the login error when clicking on start both with or without "http://" in the link. Here are some screenshots for clarity.

After entering the auth code I get this:
![1](https://cloud.githubusercontent.com/assets/22148951/18469381/4cd0be88-79a8-11e6-8221-2cde539e7004.PNG)

Then I click on close and click ok again on the auth code without changing or retyping it and I get this:
![2](https://cloud.githubusercontent.com/assets/22148951/18469380/4cd0af06-79a8-11e6-9d03-e6ca8f661850.PNG)

But then after retrieving items and clicking on start I get this:
![3](https://cloud.githubusercontent.com/assets/22148951/18469382/4cd6c38c-79a8-11e6-8de3-dcf7818f2591.PNG)

So why is it telling me I'm logged in when I'm not?

**Comment by SirMerideo** [comment]
In that case, I don't know, Sorry, I don't know how the Steam API works with all it's fancy logins.

After removing the "http" from the link it worked for me, that's all the help I can offer. Wish I could do more. 

Good luck and hope that the author will reply and/or fix it.

**Comment by Dexxxxx** [comment]
I believe I have solved the problem. After getting the first error, instead of just clicking close and ok on the old code, if I wait for the authenticator to regenerate a new code and type that one in it seems to actually log me in and I am able to search for items.

So I can now log in with a 100% success rate, but I have to enter two auth codes every time. Definitely something wrong there, but at least I have found a workaround! So yeah, closing the issue for now.
